docs/pages/common_issues/_meta.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'm not sure what this _meta.json file does.

Reply:
Its for making the Isr part uppercase in this menu:
![image](https://github.com/sst/open-next/assets/91796856/11a1037d-c74a-4b5a-b485-d7ffca32ad70)